news/2017-08-21-yuzus-new-look-contest.md

```diff
@@ -0,0 +1,77 @@
+---
+layout: post
+title: "Yuzu's New Look Contest"
+date: 2017-08-21 09:00:00 +0000
+---
+
+Intimidated by Mocha's explosive popularity, Yuzu is putting out a call for intrepid artists to help him discover his new look! Like to draw? Think you can make Yuzu shine like a diamond? Read on to find out more!
+
+[![](https://assets.ppy.sh/contests/49/header.jpg)](https://osu.ppy.sh/community/contests/49)
+
+osu!catch might be one of our smaller gamemodes, but Yuzu isn't content to let Mocha reign in the limelight forever. 
+
+It's time for a refresh, and we're trusting it to our amazingly talented community artists to make it happen!
+
+There's a few things to keep in mind, however. This contest is about giving Yuzu a **new look**, not redesigning him completely. We want to keep some things the same, but the rest is up to you!
+
+## Who is Yuzu?
+
+[![](https://osu.ppy.sh/help/wiki/Mascots/Yuzu_Profile.png)](https://osu.ppy.sh/wiki/Mascots/Gallery#yuzu)
+
+Yuzu is the _osu!catch_ mascot. He's most commonly seen in the base game skin, holding up the catcher. 
+
+The official colour for osu!catch is **blue**, and thus Yuzu's primary colour should be blue in your character design as well.
+
+His skin should be skin coloured, though. Not blue. When we say blue is his primary colour, we mean in his design, not in his skin.
+
+Yuzu is male, and a teenager. **While we're all about new looks, he needs to stay as both of those things and not magically become anything else.**
+
+## Any special features?
+
+The main theme we are aiming for with the _osu!_ mascots is **coloured bangs**. 
+
+This should be the centrepiece of your character design in some way - Yuzu's bangs should be unique and recognizable compared to pippi and Mocha.
+
+**Any entries without unique, identifiable bangs will be disqualified.** Super important to take this into consideration when considering a new design.
+
+Yuzu should always be recognizable as Yuzu regardless of what form he's in or what he's wearing, even if he's wearing nothing at all. A flashy outfit is good, but a flashier character is even better.
+
+Yuzu is also moving around a lot, perhaps the most of all the _osu!_ mascots. **Giving him features that portray motion would be an excellent idea.** Characters with long hair often have no difficulty expressing emotion. Characters with short hair usually use props or an element of their costume to achieve the same thing.
+
+## What's this about props?
+
+Props are great, as pippi and Mocha have demonstrated. Yuzu would very much like to have one as well. **You should create some sort of key prop that helps make Yuzu identifiable.**
```

Comment:
I think so? not 100% on that though, I've always used lowercase when describing her since it's a feminization of 'peppy'